commands.json

```diff
@@ -703,6 +703,7 @@
         "name": "withcount",
         "token": "WITHCOUNT",
         "type": "pure-token"
```

Comment:
hi, you missed comma here

Reply:
Yep - breaks my redis.io build :)

Reply:
Here's the fix: https://github.com/RedisBloom/RedisBloom/pull/510